app/partials/pet-details-form.php

```diff
@@ -138,7 +138,7 @@
         </p>
     </div>
 
-    <p class="h-spacing-base">
+    <p id="chip-number-wrap" class="h-spacing-base h-hide">
         <label for="chip-number">Chip number:</label>
```

Comment:
or maybe we should just set focus into the text input once you've selected it?  That might be enough